### DISCUSSION TITLE
Thickness of text

### DISCUSSION THREAD

**Discussion by Filyus** [protectwise/troika#171] [Ideas]
I think it would be cool and interesting to come up with a way to make the text thicker, but to keep the glow and borders working.

**Comment by lojjic** [comment]
Can you define "thicker"?

**Reply by Filyus** [reply]
The text is a plane now, I would like to be able to make it three-dimensional so that it has a thickness. That is, the picture is stretched along the normal. You should end up with something like a glowing neon sign.

**Reply by lojjic** [reply]
Thanks for clarifying! Some existing talk about that in this issue: https://github.com/protectwise/troika/issues/118
